### PR TITLE
New Bike Hire Docking Stations model specification 

### PR DESCRIPTION
**Pull request by sergg75**
Based on data provided by Malaga and Rennes in their open data portal. Some attributes are not available in many cases yet, and they are marked as optional.

@gtorodelvalle @jmcanterafonseca

### CODE REVIEW

File: Transportation/BikeHire/doc/spec.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+    -   Metadata:
+
+        -   `timestamp` : Timestamp of the last attribute update
+
+        -   Type: [DateTime](https://schema.org/DateTime)
+
+    -   Optional
+
+-   `outofserviceSlotNumber` : The number of slots that are out of order and
```
Comment (gtorodelvalle) [inline Transportation/BikeHire/doc/spec.md]: I don't know if it has been used as such but it looks better to me as "outOfServiceSlotNumber" :) Feel free to ignore this suggestion too ;)

File: Transportation/BikeHire/doc/spec.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+ 
+
+Data Model
+----------
+
+-   `id` : Unique identifier.
+
+-   `type` : Entity type. It must be equal to `BikeHireDockingStation`.
+
+-   `dateCreated` : Entity's creation timestamp
```
Comment (sergg75) [inline Transportation/BikeHire/doc/spec.md]: Changed

File: Transportation/BikeHire/doc/spec.md (hunk 1 above)
Comment (sergg75) [inline Transportation/BikeHire/doc/spec.md]: Changed

File: Transportation/BikeHire/doc/spec.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+    -   Optional
+
+-   `freeSlotNumber` : The number of slots available for returning and parking
+    bikes.
+
+    -   Attribute type: [Number](http://schema.org/Number)
+
+    -   Allowed values: A positive integer number, including 0. It must lower or
+        equal than `totalSpotNumber`.
```
Comment (jmcanterafonseca) [inline Transportation/BikeHire/doc/spec.md]: totalSlotNumber

File: Transportation/BikeHire/doc/spec.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+    -   Metadata:
+
+        -   `timestamp` : Timestamp of the last attribute update
+
+        -   Type: [DateTime](https://schema.org/DateTime)
+
+    -   Optional
+
+-   `outofserviceSlotNumber` : The number of slots that are out of order and
+    cannot be used to hire or park a bike.
```
Comment (jmcanterafonseca) [inline Transportation/BikeHire/doc/spec.md]: you need to say here that it It must lower or equal than `totalSlotNumber`.

File: Transportation/BikeHire/doc/spec.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+    -   Metadata:
+
+        -   `timestamp` : Timestamp of the last attribute update
+
+        -   Type: [DateTime](https://schema.org/DateTime)
+
+    -   Optional
+
+-   `availableBikeNumber` : The number of bikes available in the bike hire
+    docking station to be hired by users.
```
Comment (jmcanterafonseca) [inline Transportation/BikeHire/doc/spec.md]: It must lower or equal than `totalSpotNumber`.

File: Transportation/BikeHire/doc/spec.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+
+    -   Metadata:
+
+        -   `timestamp` : Timestamp of the last attribute update
+
+        -   Type: [DateTime](https://schema.org/DateTime)
+
+    -   Allowed values:
+
+        -   (`operative`, `outOfService`, `withIncidence`, `full`, `almostFull`)
```
Comment (jmcanterafonseca) [inline Transportation/BikeHire/doc/spec.md]: instead of `operative` perhaps it could be `working` ,

**Comment by jmcanterafonseca** [comment]
@sergg75 looks good but some improvements are needed.

I miss the `areaServed` attribute that you can find in other data models. 

By the way, once we land this we would need a JSON Schema

thanks!

File: Transportation/BikeHire/doc/spec.md (hunk 5 above)
Comment (sergg75) [inline Transportation/BikeHire/doc/spec.md]: fixed

File: Transportation/BikeHire/doc/spec.md [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+
+ 
+
+Examples of use
+---------------
+
+Bike hire docking station real time data in Malaga
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+{
```
Comment (sergg75) [inline Transportation/BikeHire/doc/spec.md]: fixed

File: Transportation/BikeHire/doc/spec.md [hunk 8]
```diff
@@ -0,0 +1,193 @@
+Bike Hire Docking Station
```
Comment (sergg75) [inline Transportation/BikeHire/doc/spec.md]: done

**Comment by sergg75** [comment]
Fixed all the issues. Comments r? @jmcanterafonseca @gtorodelvalle

File: Transportation/BikeHire/schema.json [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+				"type": "object"
+			},
+		}
+	}],
+	"required": [
+		"id",
+		"type",
+		"location"
+	]
+}
```
Comment (jmcanterafonseca) [inline Transportation/BikeHire/schema.json]: new line needed

Comment (jmcanterafonseca) [inline Transportation/BikeHire/schema.json]: this data model needs to be placed under Transportation/BikeHireDockingStation

File: Transportation/BikeHire/schema.json [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+			},
+			"contactPoint": {
+				"type": "object"
+			},
+		}
+	}],
+	"required": [
+		"id",
+		"type",
+		"location"
```
Comment (jmcanterafonseca) [inline Transportation/BikeHire/schema.json]: no other fields are mandatory?

File: Transportation/BikeHire/doc/spec.md (hunk 8 above)
Comment (jmcanterafonseca) [inline Transportation/BikeHire/doc/spec.md]: are you sure  you are using markdown format as it is being used in the specification of other data models?

**Comment by sergg75** [comment]
Fixed all the changes requested: @jmcanterafonseca

**Pull request merged**